packages/web-components/src/components/ic-top-navigation/ic-top-navigation.tsx

```diff
@@ -312,17 +314,30 @@ export class TopNavigation {
             <header role="banner">
               <div class="top-panel-container">
                 <div class="app-details-container">
-                  {hasTitle && (
-                    <a class="title-link" href={this.href}>
+                  {isSlotUsed(this.el, "router-app-title") ? (
```

Comment:
Could we remove the repeated code from the ternary if possible? Where we repeat the code for the entire icon/links.

Reply:
Yeah I saw that. I wondered if we could extract the code that's inside either the <a> or the <div> and pull it in? As everything inside the <a> is the same right? Unless I've read it wrong